src/sensors.c

```diff
@@ -96,6 +96,8 @@ static bool update_imu(void);
 // function definitions
 void init_sensors(void)
 {
+  // clear the IMU read error
+  _error_state &= ~(ERROR_IMU_NOT_RESPONDING);
```

Comment:
I made a mistake, line 254 was supposed to set the error.

Unfortunately, it gets reset in line 100 during the `imu_not_responding_error()` called in line 255.  That means it basically doesn't ever get called.   However, looking forward to our state machine setup, I hope that it gets published at least once so we know that something happened even if it gets fixed almost immediately.